### PR TITLE
remove unnecessary nop after ebreak

### PR DESCRIPTION
**Pull request by roman012285**

### CODE REVIEW

**Comment by roman012285** [comment]
seems that mini_core in rv32e configuration causing problems. I opened an issue. All other configurations works fine, I added temporary `nop` to `crt0_mini_rv32e` just for passing the PR

**Comment by roman012285** [comment]
it seems like the issue is in mini_core. I added a trk_inst to mini core and with nop and without the core executes different instructions. In the left is with nop and in the right is without nop. 
image

File: app/crt0/crt0_mini_rv32i.s
Comment (amichai-bd) [inline app/crt0/crt0_mini_rv32i.s:1]: Does this one work?

**Review by amichai-bd — APPROVED** [review]

**Comment by amichai-bd** [comment]
The issue that the ref model "finished" but keeps executing the a4 command is a known issue..
We will try and fix later

File: app/crt0/crt0_mini_rv32i.s
Comment (roman012285) [reply app/crt0/crt0_mini_rv32i.s:1]: no, I run the test only on rv32i_ref and it seems to execute the right commands so it seems the problem in the checker

**Pull request merged**